docs/source/cluster_management/sweep/targeted-sweep.rst

```diff
@@ -46,3 +46,31 @@ Targeted sweep uses a combination of :ref:`install and runtime AtlasDB configura
 
    ``enabled``, "false", "Whether targeted sweep should be run by background threads. Note that enableSweepQueueWrites must be set to true before targeted sweep can be run."
    ``shards``, "1", "Number of shards to use for persisting information to the sweep queue, enabling better parallelization of targeted sweep. The number of shards should be greater than or equal to the number of threads used for background targeted sweep. Note that this number must be monotonically increasing, and attempts to lower may be ignored. Maximum supported value is 256."
+
+Changing Sweep Strategy for a Table
+-----------------------------------
+
+.. warning::
+
+   Consult with the AtlasDB team before changing the sweep strategy of a table. Doing this incorrectly can invalidate
+   AtlasDB's correctness guarantees.
+
+Whenever targeted sweep enqueues a write into the sweep queue, it does so using the latest known sweep strategy for the
+table. If the sweep strategy for that table later changes, **targeted sweep does not recheck the strategy**, and will
+therefore sweep those entries using the old strategy.
+
+In a multinode setting, changing the sweep strategy for a table using a rolling upgrade strategy can cause some nodes to
+run read-only transactions on tables that are swept using the THOROUGH strategy by other nodes. Even with a shutdown
+upgrade, if the new strategy is CONSERVATIVE, we may still have old entries in the thorough sweep queue, and therefore
+sweep those cells using the THOROUGH strategy. Both of these cases have correctness implications for read-only
+transactions.
+
+The safe way to change the sweep strategy from THOROUGH to CONSERVATIVE:
+  1. Shut down all the nodes.
+  2. Start AtlasDB with the new table metadata, but **do not use read-only transactions on the table yet**.
+  3. Wait until targeted sweep for strategy THOROUGH has caught up to after the upgrade.
```

Comment:
Yep, this makes sense. As a user how should I do this? (i.e. I need to look at metric X and confirm that it is after my change)